src/SFA.DAS.EmployerFinance.Messages/Commands/ExpireAccountFundsCommand.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SFA.DAS.NServiceBus;
+
+namespace SFA.DAS.EmployerFinance.Messages.Commands
+{
+    public class ExpireAccountFundsCommand : Command
+    {
+        public long AccountId { get; set; }
```

Comment:
As discussed offline it might be worth including month and year properties on the command.